### DISCUSSION TITLE
PLL not locked?

### DISCUSSION THREAD

**Discussion by jruben4** [hertzg/rtl_433_docker#114] [General]
I had this docker working with unraid and a Rafael Micro R820T tuner for 433 mhz traffic.  Suddenly I am getting an error 
[R82XX] PLL not locked!

This might have happened when I updated to the newest version of unraid (6.12.10), not sure.  I tried updating the DVB drivers but that didn't change anything.

**Comment by hertzg** [comment]
You should probably ask this on the merbanan/rtl_433 repo. I am not sure what might have caused it.
